docs/source/utilities.rst

```diff
@@ -0,0 +1,72 @@
+Utilities
+===================================================
+This page will contain high-level documentation on the various utilities in
+RJ RoboCup Software.
+Utilities will be classified as anything that is not directly run after you
+launch soccer.
+
+Sourcing ros2
+---------------------------------------------------
+Run ``./source.bash`` to automatically detect whether your shell is using zsh
+or bash and source that respective setup file (they are located in ``/install``).
+Additionally, we would recommend adding ``source /opt/ros/foxy/setup.zsh`` to
+your ``.zshrc`` if using zsh or to your ``.bashrc``. So you don't have to
+type that out each time you open a new terminal. When in the future we
```

Comment:
```suggestion
type that out each time you open a new terminal. In the future, if we
```

Reply:
This will eventually have to happen. We can't stay with foxy and ubuntu 20.04 forever.

Reply:
It's still ROS2 just a different version of ROS2, I can clarify that.